build/xtask/src/config.rs

```diff
@@ -180,10 +180,14 @@ impl Config {
             .canonicalize()
             .expect("Could not canonicalize path to app TOML file");
 
+        // WARNING: Because this map provides task ID numbers directly, its
+        // contents can be used to send messages to tasks without creating a
+        // task_slot, so it's possible to deadlock. Use this with caution.
```

Comment:
The compiler won't let two tasks have task slots for each other, which is why this was needed in the first place. Otherwise, I could have done some weird stuff in the `build.rs` to generate all of the necessary slots.

Reply:
Aah I see. BTW was this comment supposed to be somewhere else. There is no map on line 186.

Reply:
Yeah I think I squashed it into the wrong commit, so when I moved it I instead duplicated it.